src/lib.rs

```diff
@@ -160,28 +160,49 @@ impl GraphicsContext {
     }
 
     /// Shows the given buffer with the given width and height on the window corresponding to this
-    /// graphics context. Panics if buffer.len() ≠ width*height. If the size of the buffer does
-    /// not match the size of the window, the buffer is drawn in the upper-left corner of the window.
+    /// graphics context.
+    ///
     /// It is recommended in most production use cases to have the buffer fill the entire window.
-    /// Use your windowing library to find the size of the window.
+    /// Use your windowing library to determine the size of the window.
+    ///
+    /// # Panics
+    ///
+    /// Panics if `buffer.len() ≠ width*height`. If the size of the buffer does not match the size of the
+    /// window.
     ///
-    /// The format of the buffer is as follows. There is one u32 in the buffer for each pixel in
-    /// the area to draw. The first entry is the upper-left most pixel. The second is one to the right
-    /// etc. (Row-major top to bottom left to right one u32 per pixel). Within each u32 the highest
-    /// order 8 bits are to be set to 0. The next highest order 8 bits are the red channel, then the
-    /// green channel, and then the blue channel in the lowest-order 8 bits. See the examples for
-    /// one way to build this format using bitwise operations.
+    /// # Pixel format
     ///
-    /// --------
+    /// The buffer layout is in a row-major layout. The buffer is drawn in the upper-left corner of
+    /// the window. The x axis grows to the right and the y axis grows downwards.
     ///
-    /// Pixel format (u32):
+    /// Each pixel is represented in memory as a [`u32`]. Starting with the most significant byte
+    /// going to the least significant byte, the byte is set to `0`. Then there is a byte for the red,
+    /// green and blue channel.
     ///
+    /// ```text
     /// 00000000RRRRRRRRGGGGGGGGBBBBBBBB
+    /// ^                              ^
+    /// MSB                          LSB
+    /// ```
+    ///
+    /// | Bit  | Channel       |
+    /// |------|---------------|
+    /// | 0    | Bit is `0`    |
```

Comment:
Isn't this `X` (the value of these bits is arbitrary and will be ignored) rather than `0`? Which isn't quite the same thing.

Reply:
The existing code on every backend should ignore them, but that isn't the same as specifying they the correct use of this library is zero.

Reply:
Maybe there could be slightly more specific wording indicating that values other than 0 may have issues with certain backends?